openapi/openapi.yaml

```diff
@@ -558,6 +558,10 @@ paths:
       - $ref: "#/components/parameters/python_version"
       - $ref: "#/components/parameters/cuda_version"
       - $ref: "#/components/parameters/image_name"
+      - $ref: "#/components/parameters/library_name"
+      - $ref: "#/components/parameters/symbol"
+      - $ref: "#/components/parameters/package_name"
+      - $ref: "#/components/parameters/rpm_package_name"
```

Comment:
These references require definition in `.components.parameters` section.

Reply:
Added now, thank you! :100: